src/v2/Apps/Components/AppShell.tsx

```diff
@@ -54,8 +54,8 @@ export const AppShell: React.FC<AppShellProps> = props => {
         </Box>
       </Box>
 
-      <Box>
-        <Box>{children}</Box>
+      <Box as="main" id="main">
+        {children}
```

Comment:
This was wrapped in two boxes for reasons that escape me. But there's no discernible reason for that, so I removed it. The `as` prop allows this to render as a `<main>` tag.